pyServer/manifests/windows/aks

```diff
@@ -19,7 +19,8 @@ copy,/Windows/System32/winevt/Logs/Microsoft-Windows-Hyper-V-Compute-Admin.evtx
 copy,/Windows/System32/winevt/Logs/Microsoft-Windows-Hyper-V-Compute-Operational.evtx
 copy,/Windows/System32/winevt/Logs/Microsoft-Windows-Host-Network-Service-Operational.evtx
 
-
+echo,### Calico Windows logs
+copy,/CalicoWindows/logs/*.log
```

Comment:
How many files are we expecting here? As it's '*'? And what is the expected total size?
Also do we know that this logs can not contain any secrets/password? If that's the case then let's add 'noscan' after it.